tox.ini

```diff
@@ -1,28 +1,33 @@
 [tox]
-envlist = py38-pylint{24}, coverage, pylint
```

Comment:
The Makefile still references py38-pylint24.

Reply:
Updated the `make test` target to run tests for all toxenvs.